src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -89,9 +89,9 @@ public App()
 		public void AssertIssue8356()
 		{
 #if __SKIA__
-			string SUT = Windows.UI.ViewManagement.ApplicationView.GetForCurrentView().Title;
-			string value = Windows.ApplicationModel.Package.Current.DisplayName;
-			Assert.AreEqual(SUT, value);
+			//string SUT = Windows.UI.ViewManagement.ApplicationView.GetForCurrentView().Title;
+			//string value = Windows.ApplicationModel.Package.Current.DisplayName;
+			//Assert.AreEqual(SUT, value);
```

Comment:
should that be removed ? or updated ?

Reply:
There's another PR that's supposed to fix this, but it's not in yet.